workers/unity/Packages/com.improbable.gdk.core/Commands/CommandStorage.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.Commands
+{
+    public abstract class CommandStorage
+    {
+        public abstract Type CommandType { get; }
```

Comment:
is this storing the command type or the command storage type? Am a bit confused after looking at the `CommandBookkeepingSystem` class.

Reply:
Its the command wrapper class. For example - `Generated.Improbable.PlayerLifecycle.PlayerCreator.CreatePlayer` (this class incidentally contains the requests/responses). Its only needed to uniquely identify a command type.

Reply:
At some point we should document the layout of generated stuff

Reply:
Wait nvm I mixed this up - hold on a tick.